modules/swagger-models/src/main/java/io/swagger/models/Path.java

```diff
@@ -49,6 +53,11 @@ public Path get(Operation get) {
         return this;
     }
 
+    public Path head(Operation get) {
```

Comment:
The function argument should be ```head``` not ```get```. This typo prevents this change from actually working.